errors/error.go

```diff
@@ -4,78 +4,124 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+
+	"github.com/gojek/fiber/protocol"
+	"google.golang.org/grpc/codes"
 )
 
-// HTTPError is used to capture the error resulting from a HTTP request
-type HTTPError struct {
+// FiberError is used to capture the error resulting from a HTTP request
+type FiberError struct {
 	Code    int    `json:"code"`
 	Message string `json:"error"`
 }
 
-// Error is a getter for the error message in a HTTPError object
-func (err HTTPError) Error() string {
+// Error is a getter for the error message in a FiberError object
+func (err FiberError) Error() string {
 	return err.Message
 }
 
-// ToJSON returns the HTTPError object as a Json encoded byte array
-func (err *HTTPError) ToJSON() ([]byte, error) {
+// ToJSON returns the FiberError object as a Json encoded byte array
+func (err *FiberError) ToJSON() ([]byte, error) {
 	return json.MarshalIndent(err, "", "  ")
```

Comment:
Unrelated to your change, but why use MarshalIndent?

Reply:
i believe its for beautifying the byte output, 

```
{ "abc": 123 }
```
```
{ 
  "abc": 123 
}
```

yea, saw it recommended in [stackoverflow](https://stackoverflow.com/questions/42788045/is-there-a-jq-wrapper-for-golang-that-can-produce-human-readable-json-output) lol

Reply:
haha, I think it will consume extra space (not much) if we pretty print it.